doc/Docker_deployment.md

```diff
@@ -24,6 +24,17 @@ Note: this has only been tested in OS X.
 9. docker pull kbase/deplbase
 
 ## Deploying a module
-
-1. docker build -t mymodulename .
-2. docker run --name mymodulename -d mymodulename
+1. generate cluster.ini and ssl
+```shell
+     git clone https://github.com/kbaseincubator/deploy_dev
+     cd deploy_dev/
+     cp site.cfg.example site.cfg
+     vi site.cfg
```

Comment:
What's this vi step about? What are they changing in site.cfg?
I think this section needs more explanation.